convert2rhel/unit_tests/systeminfo_test.py

```diff
@@ -68,26 +113,43 @@ def tearDown(self):
         if os.path.exists(unit_tests.TMP_DIR):
             shutil.rmtree(unit_tests.TMP_DIR)
 
+    @unit_tests.mock(tool_opts, "no_rpm_va", True)
+    def test_generate_modified_rpms_with_no_rpm_va(self):
+        self.assertEqual(system_info.log_modified_rpms_diff(), None)
+
+    @unit_tests.mock(tool_opts, "no_rpm_va", False)
     @unit_tests.mock(logger, "LOG_DIR", unit_tests.TMP_DIR)
-    @unit_tests.mock(utils, "run_subprocess", RunSubprocessMocked(
-        ("rpmva\n", 0)))
+    @unit_tests.mock(utils, "get_file_content", GetFileContentMocked(data=[['rpm1', 'rpm2'],
+                                                                           ['rpm1', 'rpm2']]))
+    def test_generate_modified_rpms_without_difference(self):
+        self.assertEqual(system_info.log_modified_rpms_diff(), None)
```

Comment:
No longer is dependent on `no_rpm_va`, both tests are misleading and imprecise. Test name isn't precise enough to tell me what it tests